docker-compose.yml

```
@@ -31,18 +31,19 @@ services:
       - db
     # Configura as variáveis de ambiente dentro do container
     environment:
-      MYSQL_USER: root
-      MYSQL_PASSWORD: password
-      HOSTNAME: db
-      MYSQL_PORT: 3306
+      DB_USER: 'root'
+      DB_PASSWORD: 'password'
+      DB_HOSTNAME: 'db'
+      DB_DATABASE: 'trybe_hotel'
+      DB_PORT: 3306
+      DB_DIALECT: 'mysql'
 
   # Serviço que irá rodar o banco de dados
   db:
     # Caso seu computador seja um Mac com processador Apple Silicon (M1, M2, etc), descomente a linha abaixo
     # platform: linux/x86_64
-    container_name: trybe_hotel_db
-    image: mysql:8.0.23
-    platform: linux/x86_64
+    container_name: trybe_hotel_db_2
+    image: mysql:8.0
```

Podemo manter nessa versão? É a versão que estamos usando na seção

```suggestion
    image: mysql:8.0.23
```